examples/stable-outputs-preview/index.ts

```diff
@@ -0,0 +1,49 @@
+import * as pulumi from '@pulumi/pulumi';
+import * as path from 'path';
+import * as ccapi from "@pulumi/aws-native";
+import * as aws from '@pulumi/aws';
+import { zipDirectory } from './zip';
+
+const config = new pulumi.Config();
+const desc = config.get('lambdaDescription') ?? 'test lambda';
+const role = new ccapi.iam.Role('lambda-role', {
+  assumeRolePolicyDocument: {
+    Version: "2012-10-17",
+    Statement: [
+      {
+        Effect: "Allow",
+        Principal: {
+          Service: "lambda.amazonaws.com",
+        },
+        Action: "sts:AssumeRole",
+      },
+    ],
+  },
+});
+
+
+const bucket = new ccapi.s3.Bucket('bucket');
+const object = new aws.s3.BucketObjectv2(
```

Comment:
Meta: Kind of frustrating that many rather basic use cases require layering aws classic on top of aws native in order to make it usable 
:(